### PR TITLE
Adding TIL + Minor Refactoring

### PR DESCRIPTION
**Pull request by Lundez**
- [x] Add TIL to DB
- [x] Fix Database migration
- [x] Fix Database creation (i.e. not catch error each time, but only build when required)
- [x] Add TIL-page
- [x] Update the text-gen default values to be a tiny bit better
- [x] Update smile dependency to not include huge binaries anymore
- [x] Other minor updates

### CODE REVIEW

**Comment by Lundez** [comment]
**WIP**

**Comment by Lundez** [comment]
Some images:
image
image
image
image
image

**Comment by Lundez** [comment]
@denkhan should be enough to take a look at images.
Please try to review fast ;)

File: test/ApplicationTest.kt [hunk 1]
```diff
@@ -129,9 +132,10 @@ class ApplicationTest {
             addJsonHeader()
             setBody(
                 jacksonObjectMapper()
-                    .writeValueAsString(BlogPost("Hello world", "1.2.3", "123456789", listOf("TRENDING")))
+                    .writeValueAsString(BlogPost("Hello world", "1.2.3", "123456789", listOf("TRENDING"), til = false))
             )
         }
+        println(req.response.content)
```
Comment (denkhan) [inline test/ApplicationTest.kt:138]: Here is a print :)

**Review by denkhan — APPROVED** [review]
I believe that the second image is better if we expect people to use it on mobile phones. The first one where we showcase TIL and Blogs next to eachother would probably not work

**Pull request merged**